proxy/http/HttpSM.h

```diff
@@ -679,10 +615,16 @@ HttpTransact::State::state_machine_id() const
   return state_machine->sm_id;
 }
 
+inline HttpUserAgent const &
+HttpSM::get_user_agent() const
+{
+  return _ua;
+}
+
 inline ProxyTransaction *
 HttpSM::get_ua_txn()
```

Comment:
I wondered if we should keep this because now we can do `sm->get_user_agent()->get_txn()`. Maybe we should keep it, so we can get a user-agent transaction and a server transaction in a consistent way.